### PR TITLE
fixed role select drop down

### PR DESCRIPTION
**Pull request by eparker71**

### CODE REVIEW

File: course_info/static/course_info/js/controllers/PeopleController.js [hunk 1]
```diff
@@ -43,6 +43,10 @@
                 $scope.lookup(searchTerm);
             }
         };
+        $scope.roleselected = function(role_id, role_name){
```
Comment (rascalking) [inline course_info/static/course_info/js/controllers/PeopleController.js]: style nitpicks:
* js is supposed to prefer camelCase, right?
* the other methods are of the form verbNoun(), can we stick with that?

**Comment by rascalking** [comment]
:+1:

File: course_info/static/course_info/js/controllers/PeopleController.js (hunk 1 above)
Comment (eparker71) [inline course_info/static/course_info/js/controllers/PeopleController.js]: yup, that's my bad. typo.

Comment (rascalking) [inline course_info/static/course_info/js/controllers/PeopleController.js]: Heh, I meant the params as well as the method name.  Don't sweat it, I'll tweak it while I'm doing the user lookup.

**Pull request merged**